cluster-essentials/pkg/client/install.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package client has APIs for bootstrapping tanzu cluster-essentials on a kubernetes cluster.
+package client
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"k8s.io/client-go/rest"
+)
+
+var downloadBundles = downloadBundle
```

Comment:
question: curious, why are we doing this? why not call the function directly?

Reply:
using monkey patching to mock out these api for unit test